great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -0,0 +1,212 @@
+import copy
+from typing import Any, Dict, List, Optional, Union
+
+import numpy as np
+from scipy import special
+
+import great_expectations.exceptions as ge_exceptions
+from great_expectations import DataContext
+from great_expectations.core.batch import BatchRequest
+from great_expectations.rule_based_profiler.domain_builder.domain import Domain
+from great_expectations.rule_based_profiler.parameter_builder import (
+    MultiBatchParameterBuilder,
+)
+from great_expectations.rule_based_profiler.parameter_builder.parameter_container import (
+    ParameterContainer,
+    build_parameter_container,
+    get_parameter_value,
+)
+from great_expectations.util import is_numeric
+from great_expectations.validator.validation_graph import MetricConfiguration
+from great_expectations.validator.validator import Validator
+
+NP_EPSILON: float = np.finfo(float).eps
+NP_SQRT_2: float = np.sqrt(2.0)
+
+
+class NumericMetricRangeMultiBatchParameterBuilder(MultiBatchParameterBuilder):
+    """
+    A Multi-Batch implementation for obtaining the range estimation bounds for a resolved (evaluated) numeric metric,
+    using domain_kwargs, value_kwargs, metric_name, and false_positive_rate (tolerance) as arguments.
+    """
+
+    def __init__(
+        self,
+        parameter_name: str,
+        metric_name: str,
+        metric_domain_kwargs: Optional[Union[str, dict]] = "$domain.domain_kwargs",
+        metric_value_kwargs: Optional[Union[str, dict]] = None,
+        false_positive_rate: Optional[float] = 0.0,
+        data_context: Optional[DataContext] = None,
+        batch_request: Optional[Union[BatchRequest, dict]] = None,
+    ):
+        """
+        Args:
+            parameter_name: the name of this parameter -- this is user-specified parameter name (from configuration);
+            it is not the fully-qualified parameter name; a fully-qualified parameter name must start with "$parameter."
+            and may contain one or more subsequent parts (e.g., "$parameter.<my_param_from_config>.<metric_name>").
+            data_context: DataContext
```

Comment:
Please add other parameters to docstring

Reply:
Including that the metric_name can be any numeric metric (can potentially use Jira comments)

Reply:
@anthonyburdi Done -- in next commit.

Reply:
@alexsherstinsky can you add `false_positive_rate` and `batch_request` parameters to the docstring?

Reply:
Done (in next commit).